index.html

```diff
@@ -1658,7 +1658,18 @@
 			arr[j] = temp;
 		}
 	}
-
+	
+	function getNameTokens() {
+		var tokens;
+	//	if(this.world.indexOf('9') == 1 ||this.world.indexOf('9') == 0) {
```

Comment:
'this' isn't defined here. Instead, take 'world' as an argument to the function, and pass it below, in load().